docs/zh/blog/2020-08-11-Observability-at-Scale.md

```diff
@@ -0,0 +1,99 @@
+## SkyWalking为超大规模而生
+- 作者：吴晟
+
+- 原文链接：https://www.tetrate.io/blog/observability-at-scale-skywalking-it-is/
```

Comment:
This link should include the skywalking official website link, at least.

Reply:
I have added